Dockerfile

```diff
@@ -1,12 +1,14 @@
-FROM alpine:3.5
+FROM vertigo/go-builder as builder
```

Comment:
I am aware of that. That was just a proposal, pointing to the multi-staging direction. All the rest was just copy/pasted from the original Dockerfile, I overlooked the other details.

Glad to know this has been taken care of already.